Frontend/CO.CDP.OrganisationApp/Pages/Support/OrganisationApproval.cshtml.cs

```diff
@@ -0,0 +1,47 @@
+using CO.CDP.Organisation.WebApiClient;
+using Microsoft.AspNetCore.Mvc;
+using OrganisationWebApiClient = CO.CDP.Organisation.WebApiClient;
+
+namespace CO.CDP.OrganisationApp.Pages.Support;
+
+public class OrganisationApprovalModel(
+    IOrganisationClient organisationClient,
+    ISession session) : LoggedInUserAwareModel(session)
+{
+    public OrganisationWebApiClient.Organisation? OrganisationDetails { get; set; }
+
+    [BindProperty]
+    public Boolean? Approval { get; set; }
```

Comment:
use bool rather Boolean, so we are in sync how we used in other places (common standard across code)

Reply:
This has been done